### ISSUE TITLE
A little confused on the usage

### ISSUE THREAD

**Issue by dark-swordsman** [nervous-systems/ffmpeg-opengl#7]
Hello,

I found this when trying to figure out how to apply GLSL shaders via ffmpeg. 

I was hoping to implement the shaders from [this library](https://github.com/bloc97/Anime4K), but I am unsure how to do it.

Do I just specify a value such as `-vf genericshader=/file/path/to/shader.glsl`? 

I wanted to confirm this before going through the arduous process of building ffmpeg (probably for Windows) with this.

**Comment by moe-protagonist** [comment]
No, it uses the shader defined in the source IIRC.  i had a version which did what you describe, it shouldn't be hard to implement.

**Comment by CommanderAsdasd** [comment]
> No, it uses the shader defined in the source IIRC. i had a version which did what you describe, it shouldn't be hard to implement.

If you don't mind to share it, I (and not only I) would be much appreciated.

**Comment by hooop** [comment]
Hi 👋 
It would me amazing to have some news about this project !

**Comment by Mirakud** [comment]
same please
